templates/tfengine/recipes/iam_bindings.hcl

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+schema = {
+  title                = "Recipe for iam bindings"
+  properties = {
+    state_bucket = {
+      description = "Bucket to store remote state."
+      type        = "string"
+    }
+    iam_bindings = {
+      description = "[Module](https://github.com/terraform-google-modules/terraform-google-iam)"
+      type        = "array"
+      items = {
+        type                 = "object"
+        additionalProperties = false
+        required = [
+          "parent_type",
+          "parent_id",
+          "bindings"
+        ]
+        properties = {
+          parent_type = {
+            description = "Type of the resource to assign the bindings."
+            type        = "string"
+            pattern     = "^storage_bucket|project|organization|folder|billing_account$"
+          }
+          parent_ids = {
```

Comment:
For each type you support, pleas add a comment what the parent ID should be. For example, for project, it's project ID not number, for folder, it's ...